### ISSUE TITLE
Showing device not connected when connected

### ISSUE THREAD

**Issue by Kuvimello** [fritzlb/iOS17-JIT-WIN#25]
My terminal log:
Getting bundle ID...
Got bundle ID: net.kdt.pojavlauncher.L7U6JKFQQ6
starting tunnel to device...
This might take a while. In case it freezes, either close this window and kill every python process in task manager or simply reboot your PC.
It seems like your device isn't connected. : [b'\x1b[32m2024-03-25 19:39:05\x1b[0m \x1b[35m[pc_name]\x1b[0m \x1b[34m__main__[9496]\x1b[0m \x1b[1;30mERROR\x1b[0m \x1b[31mDevice is not connected\x1b[0m\r\n']
C:\Users\[user]\DOWNLO~1\IOS17-~1\IOS17-~1>

iTunes also recognises my device

I have also tried with an apple cable

**Comment by fritzlb** [comment]
According to a few users, there might be two possible fixes: either eject the device in iTunes and reconnect after a reboot, or uninstall all drivers related to the ios device being plugged in in device manager (should be 3), then reboot and reconnect. 
I personally didn't encounter this issue yet, so it's pretty hard to debug. Could you report if any of the methods worked for you? Thank you :)

**Comment by Kuvimello** [comment]
I installed a driver and now it doesn't work on the first time but when I unplugged it and plugged it back in it then worked. I'm happy it works but it doesn't allow me to use the shortcut because I have to keep the terminal open while unplugging it.

**Comment by fritzlb** [comment]
Which driver did you install? The wetestusb?
The thing with the shortcut is super weird, idk maybe a new pymobiledevice3 bug or sth. If I have a fix, I'll report it here :)

**Comment by Kuvimello** [comment]
I tried enabling jit through a slightly different method through this reddit post ([https://www.reddit.com/r/AltStore/comments/1bfx0kt/jit_can_be_enabled_with_ios_17_on_windows_but_not/](https://www.reddit.com/r/AltStore/comments/1bfx0kt/jit_can_be_enabled_with_ios_17_on_windows_but_not/)) and it prompted me to install a usb driver so I'm not sure. I'm trying to find out which one it is.

**Comment by fritzlb** [comment]
My script should automatically prompt to install the wetestusb driver, idk what happened there/why it didn't install correctly, but yeah that driver should be correct :)
Pymobiledevice3 had a few updates in the last days, so maybe that's why. Once I find time again I might investigate that (and maybe even enable wifi support, that actually seems possible thanks to those updates right now :))

**Comment by Trojan-V** [comment]
I just encountered the same issue and after two hours of trying around, the solution for me appeared to be that your python script never prompted me if I want to install the "WeTest USB Driver" (which is mandatory for pymobiledevice3 to be able to discover connected devices).
I ran the command `python -m pymobiledevice3 bonjour` which then prompted me to install the mentioned driver. After I rebooted, the script started working fine.
